examples/pytorch/text-classification/sparseml_utils.py

```diff
@@ -0,0 +1,63 @@
+from typing import Any
```

Comment:
is there any way we can put this in a more shared location since it looks like these utilities are shared across other tasks as well?

Reply:
So far this seems to be easier because otherwise requires us to change the trainer class in the OG transformers library